src/inference/src/dev/make_tensor.cpp

```diff
@@ -197,9 +205,31 @@ class AllocatedTensor : public ViewTensor {
         }
         m_strides.clear();
         update_strides();
+
+        // TODO: Make it more accurately by handling the area that is begin changed during the reshape only
```

Comment:
let's apply these TODOs since it affects performance

Reply:
Will do it asap

Reply:
Fixed.